airflow_ai_sdk/operators/embed.py

```diff
@@ -0,0 +1,43 @@
+"""
+Module that contains the EmbedOperator class.
+"""
+
+from typing import Any
+
+from sentence_transformers import SentenceTransformer
+
+from airflow_ai_sdk.airflow import Context, _PythonDecoratedOperator
+
+
+class EmbedDecoratedOperator(_PythonDecoratedOperator):
+    """
+    Operator that builds embeddings for some text.
+    """
+
+    custom_operator_name = "@task.embed"
+
+    def __init__(
+        self,
+        op_args: list[Any],
+        op_kwargs: dict[str, Any],
+        model_name: str,
+        *args: dict[str, Any],
+        **kwargs: dict[str, Any],
+    ):
+        super().__init__(*args, op_args=op_args, op_kwargs=op_kwargs, **kwargs)
+
+        self.model_name = model_name
+
+    def execute(self, context: Context) -> list[float]:
+        print("Executing embedding")
+        text = super().execute(context)
+        if not isinstance(text, str):
+            raise TypeError("Attribute `text` must be of type `str`")
+        try:
+            model = SentenceTransformer(self.model_name)
+            embedding = model.encode(text, normalize_embeddings=True)
```

Comment:
should we let people pass `normalize_embeddings` via a param? looking into it, it's defaulted to False and I can't think of a great reason for us to change it

maybe worth more generally letting people pass in an `encode_kwargs` that gets passed here